CHANGELOG.md

```diff
@@ -1,7 +1,30 @@
 # Changelog
 
+## v0.34.0-rc6
+
+*November 5, 2020*
+
 ## v0.34.0-rc5
 
+Friendly reminder: We have a [bug bounty program](https://hackerone.com/tendermint).
+
+### IMPROVEMENTS
+
+- [statesync] [\#5516](https://github.com/tendermint/tendermint/issues/5516) Check that all heights necessary to rebuild state for a snapshot exist before adding the snapshot to the pool. (@erikgrinaker)
+
+### BUG FIXES
+
+- [blockchain/v2] [\#5499](https://github.com/tendermint/tendermint/issues/5499) Fix "duplicate block enqueued by processor" panic (@melekes)
+- [abci/grpc] [\#5520](https://github.com/tendermint/tendermint/issues/5520) Return async responses in order, to avoid mempool panics. (@erikgrinaker)
+- [blockchain/v2] [\#5530](https://github.com/tendermint/tendermint/issues/5530) Fix "processed height 4541 but expected height 4540" panic (@melekes)
+- [consensus/wal] Fix WAL autorepair by opening target WAL in read/write mode (@erikgrinaker)
+- [block] [\#5567](https://github.com/tendermint/tendermint/issues/5567) Fix MaxCommitSigBytes (@cmwaters)
+- [blockchain/v2] [\#5553](https://github.com/tendermint/tendermint/issues/5553) Make the removal of an already removed peer a noop (@melekes)
+- [evidence] [\#5574](https://github.com/tendermint/tendermint/issues/5574) Fix bug where node sends committed evidence to peer (@cmwaters)
+- [privval] [\#5583](https://github.com/tendermint/tendermint/issues/5583) Make `Vote`, `Proposal` & `PubKey` non-nullable in Responses (@marbar3778)
+- [evidence] [\#5610](https://github.com/tendermint/tendermint/issues/5610) Make it possible for abci evidence to be formed from tm evidence (@cmwaters)
```

Comment:
I believe this is a breaking change to core data structures, do we need to call it out as such?

Reply:
Good call; I'll move this into the breaking section and try to add some more color.

Reply:
oh yes, this is block protocol breaking change. This seems to also break any integrators that have implemented the changes for handling evidence from RPC. The current running testnets will need to restart as well, I believe.